### DISCUSSION TITLE
2023/08/23/%E5%86%99%E5%9C%A8%E5%89%8D%E9%9D%A2/

### DISCUSSION THREAD

**Discussion by giscus** [ch1ny/ch1ny.github.io#68] [General]
# 2023/08/23/%E5%86%99%E5%9C%A8%E5%89%8D%E9%9D%A2/

https://batucha.github.io/2023/08/23/%E5%86%99%E5%9C%A8%E5%89%8D%E9%9D%A2/

**Comment by batucha** [comment]
留言功能尝试111

**Comment by DanCoxon** [comment]
bababoi

**Reply by batucha** [reply]
bababoi x
bababoey √
